components/footer.jsx

```diff
@@ -98,16 +98,14 @@ var Footer = React.createClass({
               <ul className="list-unstyled">
                 <li><a href="https://webmaker.org/">Webmaker</a></li>
                 <li><a href="https://sendto.mozilla.org/">Donate</a></li>
-                <li><a href="">Legal</a></li>
-                <li><a href="">Privacy</a></li>
+                <li><a href="">Twitter</a></li>
               </ul>
             </div>
             <div className="col-xs-6">
               <ul className="list-unstyled">
-                <li><a href="">Contact</a></li>
-                <li><a href="">Twitter</a></li>
-                <li><a href="">Facebook</a></li>
-                <li><a href="">Partners</a></li>
+                <li><a href="">Legal</a></li>
+                <li><a href="">Privacy</a></li>
+                <li><a href="/about">About</a></li>
```

Comment:
Actually can you make this using react-router's [Link](https://github.com/rackt/react-router/blob/master/docs/api/components/Link.md) component? That's the one that actually triggers in-page transitions, and if you use the name of the route instead of an absolute path, it will also trigger compile-time errors if the route name is invalid (this isn't done w/ absolute paths, which means that it's easier for dead links to creep into our codebase).